tests/test_dash_acl.py

```diff
@@ -5,7 +5,8 @@
 
 import pytest
 
-DVS_ENV = ["HWSKU=Nvidia-MBF2H536C"]
+DVS_ENV = ["HWSKU=NPU"]
```

Comment:
NPU or DPU?

Reply:
Either one is ok, but Guohan suggested NPU.